config/software/stunnel.rb

```diff
@@ -46,7 +34,7 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
   configure_string = "./configure --with-ssl=#{install_dir}/embedded --prefix=#{install_dir}/embedded"
-  if fips_enabled
+  if fips_mode?
```

Comment:
Similar comment as above, one liner please. Also not sure why we used a `+=` vs `<<` like everywhere else, might be worth making some of that style consistent.